gpii/node_modules/flowManager/src/FlowManagerUtilities.js

```diff
@@ -43,35 +42,33 @@
         });
     };
 
+    gpii.flowManager.loadSolutionsRegistry = function (flowManager) {
+        var resolvedFolder = fluid.module.resolvePath(flowManager.options.solutionsRegistrySourceDir);
+        flowManager.options.fullSolutionsRegistry = require(resolvedFolder);
```

Comment:
This is pretty crazy - as well as illlegal : P
Options will be immutable in an upcoming version of the framework - and sticking the entire solutions registry into an option is unsuitable anyway. See comments below on returning the external view of the solutions registry to something a bit more like it used to be.